diagnostics/ENSO_RWS/doc/ENSO_RWS.rst

```diff
@@ -0,0 +1,294 @@
+Niño or La Nina events.* Predigested results are available for both El
+Niño and La Nina composites.
+
+**Version and contact information**
+
+Version 1, 03/09/2021
+
+PI: Dr. H. Annamalai (IPRC/SOEST University of/ Hawaii; hanna@hawaii.edu
+)
+
+Current developer: Jan Hafner (IPRC/SOEST University of Hawaii;
+jhafner@hawaii.edu)
+
+**Open source copyright agreement**
+
+The MDTF framework is distributed under the LGPLv3 license (see
+LICENSE.txt).
+
+**Functionality**
+
+The current package consists of the following functionalities:
+
+1. Basic ENSO diagnostics performed by script LEVEL_01.py
+
+2. Climatological flow properties script LEVEL_02.py
+
+3. Rossby wave source terms performed by script LEVEL_03.py
+
+4. Scatter plots as metrics to assess models performed by script
+   LEVEL_04.py
+
+As a module of the MDTF code package, all scripts to perform the four
+levels can be found here: ~/diagnostics/ENSO_RWS/
+
+The pre-digested observational data for model validation can be found
+here: ~/diagnostics/inputdata/obs_data/ENSO_RWS/
+
+**Required programming language and libraries**
+
+This package is coded in Python 3.8.5 and requires the following
+packages: numpy, os, math, xarray, netcdf4.
+
+The pre-processing and plotting are coded in NCAR Command Language
+Version 6.5.0.
+
+**Required model output variables and their corresponding units**
+
+The following model fields are required as monthly data:
+
+4-D variables (longitude, latitude, pressure level, time):
+
+1. *zg*: HGT geopotential height (m)
+
+2. *ua*: U wind component [m/s]
+
+3. *va*: V wind component [m/s]
+
+4. *ta*: Temperature (K)
+
+5. *wap*: Vertical velocity (Pa/s)
+
+3-D variables (longitude, latitude, time):
+
+6. *pr*: Precipitation (k/m\ :sup:`2`/s)
+
+7. *ts*: Surface temperature (K)
+
+More details can be found in the README_general.pdf document.
+
+**References**
+
+1. Annamalai, H., R. Neale and J. Hafner: ENSO-induced teleconnection:
+development of PODs to assess Rossby wave sources in climate models (in
+preparation).
+
+2. R. Neale and H. Annamalai: Rossby wave sources and ENSO- induced
+teleconnections in CAM6 model development versions and associated
+vertical processes (in preparation).
+
+   **More about ENSO_RWS**
+
+**Rossby Wave Source POD to assess ENSO-induced teleconnection
+(ENSO_RWS)**
+
+The NOAA-MDTF Rossby Wave Source (RWS) Process Oriented Diagnostic (POD)
+package would fill a critical gap in the diagnostics tools available to
+climate model developers. Both in basic-state and anomalous conditions,
+changes in the response of moist processes in model either
+parameterization modifications or tuning and calibration, can often
+change the nature of the seasonal distributions of tropical
+precipitation and associated heating, and by association moistening and
+divergence profiles. While validation of precipitation is
+straightforward, an understanding of the circulation consequences in the
+tropics, and particularly the extra-tropics is not. 
```

Comment:
tropics ~~,~~ (remove comma) and ~~particularly~~ the extra-tropics

Reply:
edited